lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -233,21 +233,23 @@ class CategoryRenderer {
     element.id = category.id;
     element.appendChild(this._renderCategoryScore(category));
 
-    const auditsGroupedByGroup = category.audits.reduce((indexed, audit) => {
+    const auditsGroupedByGroup = /** @type {!Object<string,
+        {passed: !Array<!ReportRenderer.AuditJSON>,
+        failed: !Array<!ReportRenderer.AuditJSON>}>} */ ({});
```

Comment:
eh I'm cool with splitting into passed and failed arrays instead if it makes it cleaner here